samples_project/Assets/SampleViewer/Samples/LineOfSight/readme.md

```diff
@@ -1,18 +1,29 @@
-# Line of sight
+# Determine the visibility for a line of sight
 
 Show a line of sight between two objects. Check if the line of sight is obstructed by the ArcGIS 3D object scene layer.
 
 ![Image of line of sight](LineOfSight.jpg)
 
+## How to use the sample
+
+1. Open **LineOfSight** scene.
+2. Click on **LineOfSightMap** game object in the **Hierarchy** window.
+3. Set your API key in the **Inspector** window.
+4. Click play and see the line colors changes to red if there is any object to obstruct the sight.
+
 ## How it works
 
-1. Check the box for **Mesh Colliers Enabled** on the `ArcGIS Map View` component.
-2. Create a parent object with an `ArcGIS Location` component.
-3. Create child game objects in the scene that use `Transform` components.
-4. Move an object around the scene so that it comes in and out of the **view** of another object.
-5. Add any objects that may interfere with the raycast check to the **Ignore Raycast** layer.
-6. Check for any obstructions between objects using the `Physics.Raycast` method.
-7. Use the `RaycastHit.point` property to determine where the line of sight from the first object collides.
+1. Create an ArcGIS Map with the [mesh colliders](https://developers.arcgis.com/unity/maps/mesh-collider/) enabled.
+2. Attach an [**ArcGIS Camera**](https://developers.arcgis.com/unity/maps/camera/#arcgis-camera) component to the active camera.
+3. Create a parent game object with the [**ArcGIS Location**](https://developers.arcgis.com/unity/maps/location-component/) component attached for other game object to be nested.
```

Comment:
```suggestion
3. Create a parent game object with the [**ArcGIS Location**](https://developers.arcgis.com/unity/maps/location-component/) component attached.
```

Reply:
You don't need to attach a location component to each child game object if the parent has it. And the purpose of this game object is to be a parent of other game objects that are for this sample and place them with GIS coordinates as far as see in the sample. So I think it's better for users to know the purpose of this game object.